apps/test/unit/templates/curriculumCatalog/CurriculumCatalogTest.jsx

```diff
@@ -256,3 +257,117 @@ describe('CurriculumCatalog', () => {
     });
   });
 });
+
+describe('CurriculumCatalog with url params', () => {
+  const defaultProps = {curriculaData: allCurricula, isEnglish: false};
+  let store;
+
+  beforeEach(() => {
+    store = configureStore({reducer: {responsive}});
+    store.dispatch(setResponsiveSize(ResponsiveSize.lg));
+  });
+
+  function renderWithUrlParams(urlParams) {
+    setWindowLocation({search: urlParams});
```

Comment:
it seems like bad things could happen (leaked state between tests) if you do not reset this between test runs. It doesn't look like there will be any problem if you call `resetWindowLocation` too many times., so my recommendation would be to call `resetWindowLocation` in an `afterEach` call above.